src/model-helpers/User.js

```diff
@@ -55,7 +55,7 @@ export const updateUserActivity = async (session, podUrl) => {
 
   try {
     activityDataset = await getSolidDataset(activityDocUrl, { fetch: session.fetch });
-    const activeTTLThing = getThing(activityDataset);
+    const activeTTLThing = getThing(activityDataset, `${activityDocUrl}#active`);
```

Comment:
`getThing` always needs to be called with the URL of the desired thing. Otherwise it errors.